src/hotspot/share/opto/callnode.hpp

```diff
@@ -323,9 +325,16 @@ class JVMState : public ResourceObj {
 // potential code sharing) only - conceptually it is independent of
 // the Node semantics.
 class SafePointNode : public MultiNode {
+  friend class GraphKit;
+  friend class JVMState;
   virtual bool           cmp( const Node &n ) const;
   virtual uint           size_of() const;       // Size is bigger
 
+protected:
+  JVMState* _jvms;      // Pointer to list of JVM State objects
+  void set_jvms(JVMState* s) {
+    _jvms = s;  // override const attribute in the accessor
```

Comment:
The comment should be adjusted since `_jvms` is not const anymore.

Reply:
thank you for reviewing this.   Vladimir helped me to understand the hotspot Servicability agent thing, which needs to access this field. I have to revert this change.